iothub_client/tests/global_valgrind_suppression.supp

```diff
@@ -395,4 +395,239 @@
    fun:e2e_d2c_with_svc_fault_ctrl_error_message_callback
    fun:RunTests
    fun:(below main)
+}
+{
+  OpenSSL leak
+  Memcheck:Leak
+  match-leak-kinds: definite
+  fun:malloc
+  fun:RSA_meth_dup
+  obj:*
+  obj:*
+  obj:/usr/lib/x86_64-linux-gnu/libcrypto.so.1.1
+  fun:ENGINE_ctrl_cmd_string
+  fun:ENGINE_by_id
+  obj:/usr/lib/x86_64-linux-gnu/libcrypto.so.1.1
+  fun:CONF_modules_load
+  fun:CONF_modules_load_file
+  obj:/usr/lib/x86_64-linux-gnu/libcrypto.so.1.1
+  obj:/usr/lib/x86_64-linux-gnu/libcrypto.so.1.1
+  fun:__pthread_once_slow
+  fun:CRYPTO_THREAD_run_once
+  fun:OPENSSL_init_crypto
+  fun:OPENSSL_init_ssl
+  obj:/usr/lib/x86_64-linux-gnu/libcurl.so.4.6.0
+}
+{
+  <insert_a_suppression_name_here>
```

Comment:
These are all dups. You can have one statement for all of them.

> {
  OpenSSL EC KEY leak 
  Memcheck:Leak
  match-leak-kinds: definite
  fun:malloc
  fun:CRYPTO_zalloc
  fun:EC_KEY_METHOD_new
  obj:*
  obj:*
  obj:/usr/lib/x86_64-linux-gnu/libcrypto.so.1.1
  fun:ENGINE_ctrl_cmd_string
  fun:ENGINE_by_id
  obj:/usr/lib/x86_64-linux-gnu/libcrypto.so.1.1
  fun:CONF_modules_load
  fun:CONF_modules_load_file
  obj:/usr/lib/x86_64-linux-gnu/libcrypto.so.1.1
  obj:/usr/lib/x86_64-linux-gnu/libcrypto.so.1.1
  fun:__pthread_once_slow
  fun:CRYPTO_THREAD_run_once
  fun:OPENSSL_init_crypto
  fun:OPENSSL_init_ssl
  obj:/usr/lib/x86_64-linux-gnu/libcurl.so.4.6.0
  obj:/usr/lib/x86_64-linux-gnu/libcurl.so.4.6.0
  fun:HTTPAPI_Init
  fun:HTTPAPIEX_Init
  fun:platform_init
  ...
}